app/src/ui/changes/changes-list.tsx

```diff
@@ -530,9 +530,9 @@ export class ChangesList extends React.Component<
 
   private getPlaceholderMessage(
     files: ReadonlyArray<WorkingDirectoryFileChange>,
-    singleFileCommit: boolean
+    prepopulateCommitSummary: boolean
   ) {
-    if (!singleFileCommit) {
+    if (!prepopulateCommitSummary) {
```

Comment:
Nit: this is the only place where I feel that the new change is a slight setback in terms of readability. The rest of this method body doesn't make sense unless there is only 1 file in the commit, so the original guard `if (!singleFileCommit) return` made perfect sense. Now that the condition is called `prepopulateCommitSummary`, I feel like the new name (by itself) doesn't communicate forward that this is just a more specific condition of `filesSelected.length === 1` at its core.

This is just me thinking out loud and shouldn't be taken as a blocker for this PR.  👍